.vscode/extensions.json

```diff
@@ -0,0 +1,18 @@
+{
+  // See https://go.microsoft.com/fwlink/?LinkId=827846 to learn about workspace recommendations.
+  // Extension identifier format: ${publisher}.${name}. Example: vscode.csharp
+
+  // List of extensions which should be recommended for users of this workspace.
+  "recommendations": [
+    "esbenp.prettier-vscode",
+    "eamodio.gitlens",
+    "GitHub.vscode-pull-request-github",
+    "ms-vsliveshare.vsliveshare",
+    "jock.svg",
+    "wix.vscode-import-cost",
```

Comment:
Actually, this has in the past caused annoying hangup issues for me (even on an m1 mac) when saving files, let's remove it and leave it in the doc so people can add it on their own if they want.